plugin/pkg/auth/authorizer/rbac/subject_locator.go

```diff
@@ -66,12 +70,12 @@ func (r *SubjectAccessEvaluator) AllowedSubjects(requestAttributes authorizer.At
 	}
 	errorlist := []error{}
 
-	if clusterRoleBindings, err := r.clusterRoleBindingLister.ListClusterRoleBindings(); err != nil {
+	if clusterRoleBindings, err := r.clusterRoleBindingLister.ListClusterRoleBindings(genericapirequest.NewContext()); err != nil {
```

Comment:
It's ok to have an empty context here?

Reply:
I actually wired a context in the subject locator. An empty context will cause a panic down below in the stack.